Libraries/xcsdk/Sources/SDK/Manager.cpp

```diff
@@ -123,6 +123,27 @@ executablePaths() const
     };
 }
 
+std::vector<std::string> Manager::
+executablePaths(std::vector<xcsdk::SDK::Toolchain::shared_ptr> &toolchains, xcsdk::SDK::Target::shared_ptr &target) const
+{
+    std::vector<std::string> paths;
+
+    for (Toolchain::shared_ptr const &toolchain : toolchains) {
+        std::vector<std::string> toolchainPaths = toolchain->executablePaths();
+        paths.insert(paths.end(), toolchainPaths.begin(), toolchainPaths.end());
+    }
+
+    if (target != nullptr) {
+        std::vector<std::string> targetPaths = target->executablePaths();
```

Comment:
I think this is changing behavior: before, if you used `overrideToolchains`, those paths would be used *instead* of the target's toolchains. Now, the passed-in toolchains are used *in addition* to the target's toolchains, since the target's toolchains are included in its paths. Even with that, I think the ordering is backwards: the passed-in override toolchains are now ending up *before* the target's default toolchains.